docs/design/2026-05-11-external-data-rk9.md

```
@@ -0,0 +1,705 @@
+player_count        int                 ← total VGC registrants
+has_team_lists      boolean DEFAULT false ← were team lists publicly available?
+imported_at         timestamptz DEFAULT now()
+import_status       text DEFAULT 'pending'  ← 'pending' / 'roster' / 'teams' / 'pairings' / 'complete' / 'failed'
+import_error        text
+```
+
+#### `rk9.players`
+
+Unique players across all events. Deduped by name + country (no stable external ID available).
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)